packages/fuels-contract/src/contract.rs

```diff
@@ -155,9 +156,12 @@ impl Contract {
         wallet: &LocalWallet,
         params: TxParameters,
     ) -> Result<ContractId, Error> {
-        let compiled_contract = Contract::load_sway_contract(binary_filepath).unwrap();
+        let compiled_contract = Contract::load_sway_contract(binary_filepath);
```

Comment:
would this work and be cleaner?

```suggestion
        let compiled_contract = Contract::load_sway_contract(binary_filepath)?;
        Self::deploy_loaded(&(compiled_contract), wallet, params).await
```

Reply:
That's much cleaner, thanks for the suggestion!